packages/components/slider/src/button.vue

```diff
@@ -3,7 +3,7 @@
     ref="button"
     :class="[ns.e('button-wrapper'), { hover: hovering, dragging }]"
     :style="wrapperStyle"
-    tabindex="0"
+    :tabindex="disabled ? -1 : 0"
```

Comment:
## Suggest


I think this change is great! However, looking at other UI frameworks (vuetify, mui), I found that the tooltip does not work when the slider is disabled. What do you think about adding this change?